src/rank_llm/rerank/listwise/rank_fid.py

```diff
@@ -41,6 +41,7 @@ def __init__(
         stride: int = 10,
         precision: str = "bfloat16",
         device: str = "cuda",
+        batch_size: int = 32,
```

Comment:
in line 130 it looks like the default is one, please double check running this on basilisk if 32 works I am ok with changing it to 32 for the sake of consistency with the rest